packages/eslint-plugin-office-addins/src/rules/call-sync-before-read.ts

```diff
@@ -22,6 +24,16 @@ export = {
   create: function (context: any) {
     let apiReferences: OfficeApiReference[] = [];
 
+    function checkPropertyRead(node: TSESTree.MemberExpression): boolean {
```

Comment:
I don't think the name of this function is very informative on what it does.
`checkPropertyIsRead` would make it better

Reply:
Done